www/scripts/codecheckerviewer/codecheckerviewer.js

```diff
@@ -60,7 +63,16 @@ function (declare, topic, domConstruct, Dialog, DropDownMenu, MenuItem,
 
     domConstruct.place(logo, headerPane.domNode);
 
-    //--- Menu button ---//
+    var user = CC_AUTH_SERVICE.getLoggedInUser();
+    var loginUserSpan = null
+    if (user.length > 0) {
+      loginUserSpan = domConstruct.create('span', {
+        id: 'loggedin',
+        innerHTML: "Logged in as " + user
```

Comment:
Closing `.` or a `:` somehow is missing, this sentence feels orphaned without it.

`Logged in as admin.`
`Logged in: admin.`

Something like that.